<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlatformBrowser</class>
 <widget class="QWidget" name="PlatformBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Platform Browser</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="filterText">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearFilterBn">
       <property name="text">
        <string>X</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PlatformBrowser::View" name="platformView">
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlatformBrowser::View</class>
   <extends>QTreeView</extends>
   <header>View.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/wizard.qrc"/>
 </resources>
 <connections/>
</ui>
